File: C Sharp/UriOnlineJudge/BEGINNER/_1002.cs
using System;
using System.Globalization;

namespace UriOnlineJudge.BEGINNER
{
    class _1002
    {
        public void Execute()
        {
            double pi = 3.14159;

            double r = double.Parse(Console.ReadLine());

            double a = pi * Math.Pow(r, 2);

            Console.WriteLine("A=" + a.ToString("F4", CultureInfo.InvariantCulture));
        }
    }
}

File: Go/BEGINNER/1002/1002.go
package main

import (
	"fmt"
	"math"
)

func main() {
	pi := 3.14159

	var r float64

	fmt.Scanln(&r)

	a := pi * math.Pow(r, 2)

	fmt.Printf("A=%.4f\n", a)
}
